pynest/nest/lib/hl_api_simulation.py

```diff
@@ -84,7 +83,10 @@ def Run(t):
     `Prepare` must be called before `Run` to calibrate the system, and
     `Cleanup` must be called after `Run` to close files, cleanup handles, and
     so on. After `Cleanup`, `Prepare` can and must be called before more `Run`
-    calls. Any calls to `SetStatus` between `Prepare` and `Cleanup` have
+    calls.
+
+    Any calls modifying the network, i.e., calls to `Create`, `Connect`,
+    `SetStatus` and `SetKernelStatus` between `Prepare` and `Cleanup` have
     undefined behaviour.
```

Comment:
``Disconnect`` is not mentioned there and I have honestly no idea whether it'd be safe to call it or not...

Reply:
I added it!